packages/isomorphic/test/sha256.test.ts

```diff
@@ -0,0 +1,17 @@
+import { sha256 } from '@xrplf/isomorphic/sha256'
+import { bytesToHex } from '@xrplf/isomorphic/utils'
+
+describe('sha256', () => {
+  it('hashes', () => {
+    const hashA = sha256('abc')
+    const hashB = sha256
+      .create()
+      .update(Uint8Array.from([97, 98, 99]))
```

Comment:
I thought these were using UTF-8 encoding instead of hex/ascii?

Reply:
Maybe the README should be updated / a subset should be marked as UTF-8 and a subset as hex strings?

Reply:
(This also applies to the other tests in this folder)

Reply:
Nevermind, these are using UTF-8, my quick google for the character code of `a` was incorrect. Please disregard!